### PR TITLE
Javascript update

### PR DESCRIPTION
**Pull request by Karthik-59**
Added content to JavaScript Roadmap according to issue #1838

### CODE REVIEW

File: content/roadmaps/110-java/content/100-java-fundamentals/104-datastructures.md [hunk 1]
```diff
@@ -10,5 +10,4 @@ As the name indicates itself, a **Data Structure** is a way of organizing the da
 <ResourceGroupTitle>Free Content</ResourceGroupTitle>
 <BadgeLink colorScheme='yellow' badgeText='Read' href='https://www.geeksforgeeks.org/data-structures'>What are Data Structures?</BadgeLink>
 <BadgeLink colorScheme='yellow' badgeText='Read' href='https://www.javatpoint.com/data-structure-tutorial'> Data Structures and Algorithms</BadgeLink>
-<BadgeLink badgeText='Watch' href='https://www.coursera.org/learn/data-structures'>Data Structures and Algorithms</BadgeLink>
-<BadgeLink badgeText='Watch' href='https://www.youtube.com/watch?v=9rhT3P1MDHk&list=PLkZYeFmDuaN2-KUIv-mvbjfKszIGJ4FaY'>Data Structures Illustrated</BadgeLink>
```
Comment (kamranahmedse) [inline content/roadmaps/110-java/content/100-java-fundamentals/104-datastructures.md:14]: Why is this removed?

File: content/roadmaps/110-java/content/100-java-fundamentals/104-datastructures.md [hunk 2]
```diff
@@ -10,5 +10,4 @@ As the name indicates itself, a **Data Structure** is a way of organizing the da
...
+<BadgeLink badgeText='Watch' href='https://www.coursera.org/learn/data-structures'>Data Structures and Algorithms</BadgeLink>
```
Comment (kamranahmedse) [inline content/roadmaps/110-java/content/100-java-fundamentals/104-datastructures.md:13]:
```suggestion
<BadgeLink badgeText='Watch' href='https://www.coursera.org/learn/data-structures'>Data Structures and Algorithms</BadgeLink>
<BadgeLink badgeText='Watch' href='https://www.youtube.com/watch?v=9rhT3P1MDHk&list=PLkZYeFmDuaN2-KUIv-mvbjfKszIGJ4FaY'>Data Structures Illustrated</BadgeLink>
```

File: content/roadmaps/110-java/content/100-java-fundamentals/104-datastructures.md (hunk 1 above)
Comment (Karthik-59) [reply content/roadmaps/110-java/content/100-java-fundamentals/104-datastructures.md:14]: Because it is showing conflicts.
